### PR TITLE
base-files: sysctl disable unprivileged eBPF access

### PR DESCRIPTION
**Pull request by cr6git**
https://bugzilla.redhat.com/show_bug.cgi?id=1528516

https://www.exploit-db.com/exploits/45058/

https://github.com/anthraxx/linux-hardened/commit/65fb22629f6d54ec468dd664430d72db3f911071

https://kernsec.org/wiki/index.php/Kernel_Self_Protection_Project/Recommended_Settings

### CODE REVIEW

File: srcpkgs/base-files/files/sysctl.conf [hunk 1]
```diff
@@ -19,3 +19,6 @@ kernel.kexec_load_disabled=1
 
 # Avoid non-ancestor ptrace access to running processes and their credentials.
 kernel.yama.ptrace_scope=1
+
+# Disable unprivileged eBPF access
```
Comment (Vaelatern) [inline srcpkgs/base-files/files/sysctl.conf]: Useless comment.

Comment (Vaelatern) [reply srcpkgs/base-files/files/sysctl.conf]: Something better might link to the issues.

**Comment by Vaelatern** [comment]
Only a problem through 4.14.8 - we only have 3.16, 3.18, 4.4, 4.9 that are vulnerable. While that is 4 kernels, even our -lts pointer is safe from the exploit.

**Comment by Vaelatern** [comment]
This update won't reach people who don't update, and the only real target are those who update everything except the kernel.

**Comment by atweiden** [comment]
Related to https://kernsec.org/wiki/index.php/Kernel_Self_Protection_Project/Recommended_Settings, what about disabling `CONFIG_MODIFY_LDT_SYSCALL` in [x86_64-dotconfig](https://github.com/void-linux/void-packages/blob/31e9f75211b765321c37707d27db34ff55c6456e/srcpkgs/linux4.18/files/x86_64-dotconfig#L685)?

```diff
-CONFIG_MODIFY_LDT_SYSCALL=y
+# CONFIG_MODIFY_LDT_SYSCALL is not set
```

**Comment by newbluemoon** [comment]
@atweiden It was reenabled in 79fdce105d4c74f088c3327b24a0244584a21046
> Reenable CONFIG_MODIFY_LDT_SYSCALL=y to build i686 SBCL and Go 1.4.

**Comment by Vaelatern** [comment]
So really only an issue on our 4.9?

**Comment by cr6git** [comment]
Goodbye.

**Pull request closed without merging**